Xwt/Xwt/CellView.cs

```diff
@@ -165,6 +166,8 @@ public void OnButtonReleased (ButtonEventArgs args)
 			set { visible = value; }
 		}
 
+		public AccessibleFields AccessibleFields { get; set; }
```

Comment:
Please change to:

```c#
AccessibleFields accessibleFields;

public AccessibleFields AccessibleFields {
    get {
        if (accessibleFields == null)
            accessibleFields = new AccessibleFields ();
        return accessibleFields;
    }
}
```

Reply:
@slluis I would also need to remove these checks:
https://github.com/mono/xwt/pull/897/files#diff-caf37f9c62d1d45f8d30ad776d8b62d7R196
https://github.com/mono/xwt/pull/897/files#diff-2229d0433bd649eface587819a9c0a4cR190

Which would result in executing all the code in these methods even if the `AccessibilityFields` wasn't initialized.
Do you think it is still worth doing this change?

Reply:
Yes, that change is necessary. ICellViewFrontend.AccessibleFields could still return null if the accessibleFields field has no been initialized.

Reply:
👌